test/service/test_jobs.py

```diff
@@ -73,46 +73,46 @@ def create_mock(self, target_name):
         return patcher.start()
 
     def test_returns_job(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
```

Comment:
`pzsvc-ndwi-py` is the default value for the `create_algorithm` function.  You don't need to repeat it here.